assets/javascripts/discourse/components/modal/discourse-post-event-invitees.js

```diff
@@ -0,0 +1,65 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import { debounce } from "discourse-common/utils/decorators";
+
+export default class DiscoursePostEventInvitees extends Component {
+  @service store;
+
+  @tracked invitees;
+  @tracked filter;
+  @tracked isLoading = false;
+  @tracked type = "going";
+
+  constructor() {
+    super(...arguments);
+    this._fetchInvitees();
+  }
+
+  get title() {
+    return `discourse_post_event.invitees_modal.${
+      this.args.model.params?.title || "title_invited"
+    }`;
+  }
```

Comment:
minor change but I'd use `I18n.t()` here, so that the getter would actually return the title (so the name would match the impl) and not title's i18n path